src/shogun/clustering/KMeansBase.cpp

```diff
@@ -77,12 +76,12 @@ void KMeansBase::set_random_centers()
 		SGVector<float64_t> vec=lhs->get_feature_vector(cluster_center_i);
 
 		for (int32_t j=0; j<dimensions; j++)
-			mus(j,i)=vec[j];
+			cluster_centers(j, i) = vec[j];
```

Comment:
no loop necessary here, maybe touching it now is an opportunity to either use sg_memset or add a function to linalg?